src/dotnet/APIView/APIViewWeb/Pages/Assemblies/RequestedReviews.cshtml

```diff
@@ -0,0 +1,144 @@
+﻿@page "RequestedReviews"
+@model APIViewWeb.Pages.Assemblies.RequestedReviews
+@using APIViewWeb.Helpers
+@using APIViewWeb.Models
+@{
+    ViewData["Title"] = "Requested Reviews";
+    var userPreference = PageModelHelpers.GetUserPreference(Model._preferenceCache, User.GetGitHubLogin()) ?? new UserPreferenceModel();
+    ViewData["UserPreference"] = userPreference;
```

Comment:
This is now TempData https://github.com/Azure/azure-sdk-tools/blob/ec2efc121a4fab4d2b03b860dcec98907f6f7d6c/src/dotnet/APIView/APIViewWeb/Pages/Shared/_Layout.cshtml#L43